src/index.html

```diff
@@ -13,10 +14,53 @@
     <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;
+    0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="./images/logo.png">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
```

Comment:
The `alt` attribute should describe the image content. Instead of 'logo', provide a more descriptive alternative text.